Source/WebKit/Shared/API/Cocoa/_WKRemoteObjectRegistry.mm

```diff
@@ -175,10 +175,11 @@ - (void)_sendInvocation:(NSInvocation *)invocation interface:(_WKRemoteObjectInt
 
     [encoder encodeObject:invocation forKey:invocationKey];
 
-    if (!_remoteObjectRegistry)
+    RefPtr protectedRemoteObjectRegistry = _remoteObjectRegistry;
```

Comment:
I think we can just name this `remoteObjectRegistry`. We avoid `protected` in local variable names as much as possible nowadays.